crawl-ref/source/wiz-dgn.h

```diff
@@ -28,7 +29,8 @@ void wizard_interlevel_travel();
 void wizard_list_levels();
 void wizard_recreate_level();
 void wizard_clear_used_vaults();
-bool debug_make_trap(const coord_def& pos = you.pos());
+bool debug_make_trap(const coord_def& pos = you.pos(),
+                     dungeon_feature_type feat = DNGN_FLOOR);
```

Comment:
Could this parameter be `trap_type trap = TRAP_UNASSIGNED` instead of using `dungeon_feature_type`?